### DISCUSSION TITLE
Birlikte Proje Geliştirerek Öğrenme

### DISCUSSION THREAD

**Discussion by ercan1s1k** [community/community#136429] [Discussions]
### Select Topic Area

General

### Body

Arkadaşlar bildiğiniz gibi tek başımıza kod yazıp durmak bir yere kadar. Birbirimizin bilgi ve deneyimlerinden faydalanmak, bizi geliştirir. Bu nedenle şimdiye kadar bir başkasıyla birlikte yazılım geliştirmedim. şimdilik html ve css ağırlıklı yani web tasarım ağırlıklı çalıştım. çünkü javascript'te girince çatallaşıp yani dallanıp vuejs reactjs react native es6 es7 gibi dallandığını gördüm ve hangisine yöneleceğimi bilmediğimden sadece html ve css'e odaklandım şimdilik. hangi js kütüphane veya framework'ü ile çalışmam üstüne bana önerilerde bulunabilirsiniz. Tabi ikna edici bir açıklama veya sebeplerle... :) Bu arada github respository private olunca, linki başkasına iletilince o başkası respository'mize erişebiliyor diye biliyorum. Ama birlikte nasıl aynı respository üzerinde çalışabileceğimi bilmiyorum..:) Bu konuda da da yardımınızı esirgemeyin ^_^

**Comment by LiteBrite82** [comment]
Hi @ercan1s1k , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
